src/main/java/service/impl/CsvFileReaderServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import service.CsvFileReaderService;
+
+public class CsvFileReaderServiceImpl implements CsvFileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        try (var lines = Files.lines(Paths.get(filePath))) {
+            return lines
+                    .skip(1)

```

Comment:
```suggestion
                    .skip(OFFSET)
```
let's make it more visible